### PR TITLE
feat: OV inference for SmolVLA

### PR DESCRIPTION
**Pull request by sovrasov**
# Pull Request

## Description

- Add OV inference for SmolVLA policy
- Introduce related numpy-based preprocessors
- Add tokenizer support to OV inference adapter and `InferenceModel`
- Update e2e to conduct OV export
- Bugfixes in existing preprocessing and Observation

## Type of Change

- [x] ✨ `feat` - New feature
- [x] 🧪 `test` - Tests

## Examples
```python
l_dm = LeRobotDataModule(repo_id=data_path, train_batch_size=1, episodes=[0])
policy = SmolVLA(dataset_stats=l_dm.train_dataset.stats).eval()
policy.to_openvino("trained_model_ov")

inference_model = InferenceModel("trained_model_ov")
obs = next(iter(l_dm.train_dataloader()))
result = inf_model.select_action(obs.to_numpy().to_dict())
```

## Known issues

2% relative difference vs torch when predicting on Libero.

### CODE REVIEW

File: library/src/physicalai/inference/adapters/base.py [hunk 1]
```diff
@@ -82,6 +82,29 @@ def default_device(self) -> str:  # noqa: PLR6301
         """
         return "cpu"
 
+    def load_tokenizer(self, tokenizer_path: Path) -> None:  # noqa: PLR6301, ARG002
```
[the remark this answers is not in the document]
Comment (sovrasov) [reply library/src/physicalai/inference/adapters/base.py]: Runner is also an idea here. But should it use the same adapter to invoke tokenizer?

File: library/src/physicalai/inference/preprocessors/smolvla.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+import cv2
+import numpy as np
+
+from physicalai.inference.constants import IMAGE_MASKS, IMAGES
+
+from .base import Preprocessor
+
+
+class ResizeSmolVLA(Preprocessor):
```
Comment (eugene123tw) [inline library/src/physicalai/inference/preprocessors/smolvla.py]: Is it possible that we decompose `ResizeSmolVLA` into reusable primitives?

`ResizeSmolVLA` bundles three independent operations — resize-with-pad, pixel normalization, and image collection/masking — into a single SmolVLA-specific class. These operations are generic and already needed by Pi05 (which reimplements them with slightly different parameters in `_resize_with_pad_torch` / `_preprocess_images`).

Consider splitting into three focused preprocessors:

| Preprocessor | Responsibility | Parameters |
|---|---|---|
| `ImageResize` | Aspect-ratio resize + pad | `image_resolution`, `pad_mode` (`"top_left"` \| `"center"`), `pad_value`, `interpolation` |
| `ImageNormalize` | Pixel range transform | `mode` (`"to_bipolar"` maps [0,1]→[-1,1]) |
| `ImageCollect` | Gather `images.*` keys → stacked `IMAGES` + `IMAGE_MASKS` | — |

**SmolVLA pipeline (current → proposed):**
```python
# Current: single monolithic preprocessor
preproc_specs = [
    ComponentSpec(type="smolvla_resize", init_args={"image_resolution": (512, 512)}),
    ComponentSpec(type="new_line", init_args={}),
]

# Proposed: composable primitives
preproc_specs = [
    ComponentSpec(type="image_resize", init_args={"image_resolution": (512, 512), "pad_mode": "top_left"}),
    ComponentSpec(type="image_normalize", init_args={"mode": "to_bipolar"}),
    ComponentSpec(type="image_collect", init_args={}),
    ComponentSpec(type="new_line", init_args={}),
]
```

**Pi05 pipeline (currently hardcoded in torch preprocessor → reuses the same components):**
```python
preproc_specs = [
    ComponentSpec(type="image_resize", init_args={"image_resolution": (224, 224), "pad_mode": "center"}),
    ComponentSpec(type="image_normalize", init_args={"mode": "to_bipolar"}),
    ComponentSpec(type="image_collect", init_args={}),
]
```


The only difference between SmolVLA and Pi05 image preprocessing is `pad_mode` and `image_resolution`. Today that's two completely separate implementations (numpy in `smolvla.py`, torch in `pi05/preprocessor.py`). With decomposition, both models compose the same building blocks with different config.
And, any new VLA policy that needs resize+normalize+collect picks from existing components rather than writing another `Resize<ModelName>` class.

**Review by eugene123tw — APPROVED** [review]
